x-pack/legacy/plugins/code/index.ts

```diff
@@ -61,6 +62,9 @@ export const code = (kibana: any) =>
         ui: Joi.object({
           enabled: Joi.boolean().default(true),
         }).default(),
+        integrations: Joi.object({
```

Comment:
please rebase because of this change: https://github.com/elastic/kibana/pull/46664. For this flag, just leave here as the `xpack.code.integrations.enabled` flag does.

Reply:
This was rebased onto master; it wasn't obvious what needed to change and there were no conflicts, so nothing really changed here. Please give this another look.

Reply:
It should be OK then.